server/routes/login.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+
+const login_service = require("../../services/user.services.js");
```

Comment:
```suggestion
const loginService = require("../../services/user.services.js");
```
En chamelCase c'est mieux 😎